packages/checkout/widgets-lib/src/widgets/sale/functions/sortAndDeduplicateCurrencies.test.ts

```diff
@@ -0,0 +1,197 @@
+import { SaleWidgetCurrency, SaleWidgetCurrencyType } from '../types';
+import { sortAndDeduplicateCurrencies } from './sortAndDeduplicateCurrencies';
+
+describe('sortAndDeduplicateCurrencies', () => {
+  it('sortAndDeduplicateCurrencies', () => {
```

Comment:
best to describe what being tested
`it('should remove duplicates and sort currencies by: base -> settlement -> swappable')`